<ui version="4.0" >
 <class>config</class>
 <widget class="QWidget" name="config" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>410</height>
   </rect>
  </property>
  <property name="styleSheet" >
   <string/>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QCheckBox" name="shadowCheckBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Shadow</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="frameCheckBox" >
       <property name="text" >
        <string>Set a frame around the picture</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Change frame color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="KColorButton" name="changeFrameColor" >
        <property name="color" >
         <color>
          <red>70</red>
          <green>90</green>
          <blue>130</blue>
         </color>
        </property>
        <property name="defaultColor" >
         <color>
          <red>70</red>
          <green>90</green>
          <blue>130</blue>
         </color>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Look</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QRadioButton" name="squareButton" >
        <property name="text" >
         <string>Squared Corners</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="roundButton" >
        <property name="text" >
         <string>Round Corners</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QComboBox" name="pictureComboBox" >
        <item>
         <property name="text" >
          <string>Static Picture</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>SlideShow</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QStackedWidget" name="stackedWidget" >
        <property name="minimumSize" >
         <size>
          <width>350</width>
          <height>51</height>
         </size>
        </property>
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <widget class="QWidget" name="page" >
         <layout class="QHBoxLayout" >
          <item>
           <widget class="KUrlComboRequester" name="picRequester" >
            <property name="acceptDrops" >
             <bool>true</bool>
            </property>
            <property name="filter" >
             <string>*.png *.jpeg *.jpg</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3" >
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <layout class="QVBoxLayout" >
            <item>
             <widget class="KUrlRequester" name="slideShowRequester" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="filter" >
               <string>*.png *.jpeg *.jpg</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="label_2" >
                <property name="text" >
                 <string>Change images every:</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>271</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QTimeEdit" name="TimeSpinner" >
                <property name="minimumTime" >
                 <time>
                  <hour>0</hour>
                  <minute>0</minute>
                  <second>0</second>
                 </time>
                </property>
                <property name="currentSection" >
                 <enum>QDateTimeEdit::HourSection</enum>
                </property>
                <property name="displayFormat" >
                 <string>h:mm:ss</string>
                </property>
                <property name="time" >
                 <time>
                  <hour>0</hour>
                  <minute>0</minute>
                  <second>0</second>
                 </time>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlComboRequester</class>
   <extends>KUrlRequester</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>frameCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>changeFrameColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>469</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel" >
     <x>166</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pictureComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>184</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel" >
     <x>184</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
